.pre-commit-config.yaml

```diff
@@ -20,13 +20,3 @@ repos:
     hooks:
       - id: yamllint
         files: ^.*\.(yml|yaml)$
-
-  - repo: https://github.com/asottile/seed-isort-config
-    rev: v2.2.0
-    hooks:
-      - id: seed-isort-config
-
-  - repo: https://github.com/pre-commit/mirrors-isort
-    rev: v5.10.1
-    hooks:
-      - id: isort
```

Comment:
I think we want to keep isort, just remove `seed-isort-config`, which is the thing that fills in `known_third_party`. We still want to keep isort running it's checks.